ember-app/app/models/new/issue.js

```diff
@@ -44,6 +44,7 @@ var Issue = Model.extend({
   },
   updateLabels : function () {
     this.set("processing", true);
+    this.set("labels", this.data.other_labels);
```

Comment:
issue.labels is all the labels that are on the issue

issue.other_labels is the labels minus the HuBoard labels

so I'm concerned that you are going to affect the column label later down the road by doing that